modules/dnn/src/net_impl_fuse.cpp

```diff
@@ -209,8 +209,9 @@ void Net::Impl::fuseLayers(const std::vector<LayerPin>& blobsToKeep_)
 
                 if (!nextData->params.has("operation") || toLowerCase(nextData->params.get<String>("operation")) != "add")
                 {
+                    String operation_text = toLowerCase(nextData->params.get<String>("operation", "sum"));
                     CV_LOG_DEBUG(NULL, "DNN/CPU: fusion with NaryEltwise or Eltwise Layer operation is not supported: "
-                        << nextData->params.get<String>("operation"));
+                        << operation_text);
```

Comment:
Some compilation modes may ignore `CV_LOG_DEBUG` statement completely (empty macro).
We should NOT create orphaned variables which are not used (and we don't want to evaluate unused expressions and call functions).